internal/pkg/service/common/configmap/dump.go

```diff
@@ -43,10 +43,13 @@ func (v dumpedValue) MarshalJSON() ([]byte, error) {
 }
 
 // Dump the configuration structure to an internal buffer.
-func (d *Dumper) Dump(v ConfigStruct) *Dumper {
+func (d *Dumper) Dump(v any) *Dumper {
 	err := Visit(reflect.ValueOf(v), VisitConfig{
 		OnField: mapAndFilterField(),
 		OnValue: func(vc *VisitContext) error {
+			if !vc.Leaf {
+				return nil
```

Comment:
Aby to bolo mozne implementovat, tak potrebujem drobnu zmenu vo `Visit`.

Doteraz sa `OnValue` callback volal iba na koncových, `leaf` nodes.

Teraz sa zavolá aj na všetkých hodnotách "po ceste".

Reply:
`Dump` / `StructToFlags` funkcie funguju bez zmeny, preto som tam doplnil podmienku.